tcf_website/static/reviews/new_review.js

```diff
@@ -28,15 +29,16 @@ jQuery(function($) {
             }).appendTo("#subject");
         });
         return this;
-    });
-
-    // Enable subject selector, disable the following
-    $("#subject").prop("disabled", false);
-    $("#course").prop("disabled", true);
-    $("#instructor").prop("disabled", true);
-    $("#semester").prop("disabled", true);
+    })
+        .done(function() { // Second callback
+            // Enable subject selector, disable the following
+            $("#subject").prop("disabled", false);
+            $("#course").prop("disabled", true);
+            $("#instructor").prop("disabled", true);
+            $("#semester").prop("disabled", true);
+        });
```

Comment:
Are these edits related to this PR? You might want to pull these changes into a different one if not and I can review it there
If this is for autopopulating the create review form, I'm unable to get it to work

Reply:
Also could we make all of them disabled by default in the HTML and enable the ones needed instead

Reply:
It's a minor fix for a edge case bug that @alex-shen1 found that I can't reproduce (clicking on a subsequent dropdown before API call finishes will show no options). I'm putting the dropdown enable code in the callback to fix that. Had to use the same logic for the new JQuery code for ths course-instructor page, so why not. Are you able to test this for me at all?

Reply:
I can disable everything by default in the HTML, but I'll still need this logic here if the user decides to backtrack (switching subdepartment disables everything after course, etc.)

Reply:
I'll pull this out into a different PR actually
also going to address some of your other review form related feature requests